sdk/core/core-client-paging-rest/package.json

```diff
@@ -11,31 +11,28 @@
   },
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:browser": "npm run build:ts && cross-env ONLY_BROWSER=true rollup -c 2>&1",
-    "build:node": "npm run build:ts && cross-env ONLY_NODE=true rollup -c 2>&1",
-    "build:samples": "echo Skipped.",
+    "build:samples": "echo Obsolete",
     "build:test": "tsc -p . && rollup -c 2>&1",
-    "build:ts": "tsc -p .",
-    "build": "npm run build:ts && rollup -c 2>&1 && api-extractor run --local",
-    "check-format": "prettier --list-different \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "clean": "rimraf dist dist-* types *.tgz *.log",
+    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* temp types *.tgz *.log",
+    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --stripInternal --mode file --out ./dist/docs ./src",
     "execute:samples": "echo skipped",
-    "extract-api": "npm run build:ts && api-extractor run --local",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
     "lint": "eslint package.json api-extractor.json src test --ext .ts",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
-    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
-    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
-    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run clean && tsc -p . && npm run unit-test:node && npm run integration-test:node",
```

Comment:
test:node command is meant to be for convenience(for local development).

Most of the packages have unit-test and integration-test exactly the same.
There is no point in running them both.

I recommend putting one of them - whichever is being used - unit-test:node in this case, since integration-test:node is skipped.

Reply:
I think by interpreting the name `test:node` you would expect it to run all tests for node whether they're unit or integration. I do not see what makes it inconvenient now? if integration tests are skipped, then the script will run just the unit tests which is what I think you're looking for here.